client/state/plans/get-interval-type-for-term.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
```

Comment:
I don't see a good reason for this function to be in `state` at all - it doesn't use the state at all. Should it be in `/lib/plans` instead?